vector/src/main/res/values/strings.xml

```diff
@@ -3094,4 +3094,8 @@
     <string name="live_location_labs_promotion_description">Please note: this is a labs feature using a temporary implementation. This means you will not be able to delete your location history, and advanced users will be able to see your location history even after you stop sharing your live location with this room.</string>
     <string name="live_location_labs_promotion_switch_title">Enable location sharing</string>
 
+    <plurals name="room_removed_messages">
+        <item quantity="one">%d message removed</item>
+        <item quantity="other">%d messages removed</item>
```

Comment:
When expanded, we can see the wording "Message deleted". So this is not coherent with this new wording (deleted/removed).
If we want to keep "%d messages removed", maybe we could change the other wording to "Message removed"?
The action to redact a message is "Remove..." so I think we should change the other wording.

Also, FTR (but I do not call for a change in this PR) the message on Element Web is:

<img width="221" alt="image" src="https://user-images.githubusercontent.com/3940906/178358335-4da8d1a3-d0b8-4d22-81da-5dc5686b1014.png">

Reply:
I agree we should change to the same syntax "remove". But I guess we should create a new String for that and we cannot update the english translation of the existing one? And I am concerned about the synchronization of this change with the other platforms if we do such a change.

Also, the difference with the merged message with Web is explained by the fact we may want message to be shorter in mobile so I am guessing that is why we do not repeat the author name of the message. The authors are already visible through avatar icons.

Reply:
We can update the English string if the meaning is close. This is the case here. Translators will see the the string has been change and may update the translations. Synchronization is important, but wording coherency in a single client is more important IMHO. We may discuss that with other teams.

Reply:
Agree. Updated in [6eb05f8](https://github.com/vector-im/element-android/pull/6501/commits/6eb05f8eb51b7c6afa3505a1ab736d9b4be15353)